packages/main/src/RPA/Excel/Files.py

```diff
@@ -198,7 +201,9 @@ def _load_workbook(self, path, data_only):
             "verify that the path and extension are correct"
         )
 
-    def create_workbook(self, path=None, fmt="xlsx"):
+    def create_workbook(
+        self, path: Optional[str] = None, fmt: Optional[str] = "xlsx"
```

Comment:
```suggestion
        self, path: Optional[str] = None, fmt: str = "xlsx"
```
wouldn't break unexpectedly if you pass `fmt=None` (will raise `ValueError(f"Unknown format: None")`) but also I see no point on marking it as `Optional` since it's not like you're getting any default good behaviour if you send a `None` to this param. (remember, there's a [diff](https://docs.python.org/3/library/typing.html#typing.Optional) between optional parameters (keyword arguments) and `Optional` values)

> Note that this is not the same concept as an optional argument, which is one that has a default. An optional argument with a default does not require the Optional qualifier on its type annotation just because it is optional. For example: